src/Microsoft.AspNetCore.Authentication.Twitter/Events/TwitterCreatingTicketContext.cs

```diff
@@ -11,37 +11,38 @@ namespace Microsoft.AspNetCore.Authentication.Twitter
     /// <summary>
     /// Contains information about the login session as well as the user <see cref="System.Security.Claims.ClaimsIdentity"/>.
     /// </summary>
-    public class TwitterCreatingTicketContext : BaseTwitterContext
+    public class TwitterCreatingTicketContext : AuthenticateResultContext<TwitterOptions>
     {
         /// <summary>
         /// Initializes a <see cref="TwitterCreatingTicketContext"/>
         /// </summary>
         /// <param name="context">The HTTP environment</param>
         /// <param name="scheme">The scheme data</param>
         /// <param name="options">The options for Twitter</param>
+        /// <param name="ticket">AuthenticationTicket.</param>
         /// <param name="userId">Twitter user ID</param>
         /// <param name="screenName">Twitter screen name</param>
         /// <param name="accessToken">Twitter access token</param>
         /// <param name="accessTokenSecret">Twitter access token secret</param>
         /// <param name="user">User details</param>
-        /// <param name="properties">AuthenticationProperties.</param>
         public TwitterCreatingTicketContext(
             HttpContext context,
```

Comment:
@Tratcher nah, discard my remark. This pattern won't play well with the OAuth2 handler anyway.